policy/release/buildah_build_task_test.rego

```diff
@@ -0,0 +1,49 @@
+package policy.release.buildah_build_task
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+
+test_good_dockerfile_param if {
+	lib.assert_equal(set(), deny) with input.attestations as [_attestation({"parameters": {"DOCKERFILE": "./Dockerfile"}})] with data["task-bundles"] as lib.bundles.bundle_data
+}
+
+test_dockerfile_param_https_source if {
+	expected := {{
+		"code": "dockerfile_param_external_source",
+		"effective_on": "2022-01-01T00:00:00Z",
+		"msg": "DOCKERFILE param value (https://Dockerfile) is an external source",
+	}}
+	lib.assert_equal(expected, deny) with input.attestations as [_attestation({"parameters": {"DOCKERFILE": "https://Dockerfile"}})] with data["task-bundles"] as lib.bundles.bundle_data
+}
+
+test_dockerfile_param_http_source if {
+	expected := {{
+		"code": "dockerfile_param_external_source",
+		"effective_on": "2022-01-01T00:00:00Z",
+		"msg": "DOCKERFILE param value (http://Dockerfile) is an external source",
+	}}
+	lib.assert_equal(expected, deny) with input.attestations as [_attestation({"parameters": {"DOCKERFILE": "http://Dockerfile"}})] with data["task-bundles"] as lib.bundles.bundle_data
+}
+
+test_dockerfile_param_not_included if {
+	expected := {{
+		"code": "dockerfile_param_not_included",
+		"effective_on": "2022-01-01T00:00:00Z",
+		"msg": "DOCKERFILE param is not included in the task",
+	}}
+	lib.assert_equal(expected, deny) with input.attestations as [_attestation({})]
+}
+
+_attestation(params) = attestation if {
+	attestation := {"predicate": {
+		"buildType": lib.pipelinerun_att_build_types[0],
+		"buildConfig": {"tasks": [{
+			"name": "buildah",
```

Comment:
Nit: It would be great to name this something other than "buildah", so it becomes obvious if we do break the implementation.

Reply:
Something like this? https://github.com/hacbs-contract/ec-policies/pull/218/commits/7f4534201ff63bca30bfbf29e72cfced840f0663

Reply:
Closer. But because `_attestation` is always called with `'buildah'` as the parameter, it still produces something that has the same value for `.name` and `.ref.name`. Just change the `.name` to something like `"ignored"` (the line I commented on)